classes/app.rb

```diff
@@ -0,0 +1,136 @@
+require_relative 'person'
+require_relative 'decorator'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'There are no books in the library'
+    else
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}".capitalize
+      end
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      puts 'There are no people in the library'
+    else
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, Age: #{person.age}, ID: #{person.id}"
+      end
+    end
+  end
```

Comment:
- [ ] I'm having some errors trying to list all people, and it's because you're calling the method `list_all_people` from your `main.rb` file but the name of this method is `list_people`. Please consider fixing this typo! 🤓 🔧 

![image](https://user-images.githubusercontent.com/77069011/233806084-efd959ed-a0fe-40d5-97ae-4a944f93ff7e.png)

- [ ] Also, when people get listed, there's an error displaying the correct data:

![image](https://user-images.githubusercontent.com/77069011/233806119-cf244a44-5ecd-4e67-b01d-b59e9d8e6951.png)

As you can see the `Age` attribute is displaying the `Classroom`, please fix this! 🤓 🔧 

- [ ] Something similar happens when I the person is a `Teacher`:

![image](https://user-images.githubusercontent.com/77069011/233806212-4df6ec70-06df-467a-81de-f2297853da51.png)

Please fix those errors! 🤓 🔧